packages/pages/src/components/image/image.tsx

```diff
@@ -22,24 +42,78 @@ export const Image = ({
     }
   }, []);
 
-  if (layout != "fixed" && (width || height)) {
+  if (layout != ImageLayout.FIXED && (width || height)) {
     console.warn(
       "Width or height is passed in but layout is not fixed. These will have no impact. If you want to have a fixed height or width then set layout to fixed."
     );
   }
 
-  const imgWidth: number = image.image.width;
-  const imgHeight: number = image.image.height;
-  const imgUUID: string = image.image.url.split("/")[4];
+  const imgWidth: number = image.width;
+  const imgHeight: number = image.height;
+  const imgUrlValid = !!image.url && image.url.split("/").length > 4;
```

Comment:
Turn this validation into its own function. I also think we should validate it more that just checking there are 4 slashes. We should also check our mktgcdn domain.